src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  const filteredMen
+    = century && men.filter(person => Math.ceil(person.died / 100) === century);
```

Comment:
You can simplify the filtering logic by combining it with the initial 'men' filtering. Use a single filter method with a condition that checks both if the person is male and if their century of death matches the given century.